nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/realm/ldap/LDAPRealm.java

```diff
@@ -117,6 +118,8 @@
  */
 @Service
 public final class LDAPRealm extends BaseRealm {
+    // Child logger of javax.enterprise.system.core.security 
+    protected static final Logger _groupSearchLogger = Logger.getLogger(_logger.getName() + ".ldaprealm.groupsearch");
```

Comment:
Not sure about our naming guide but I'd think we do not use underscore first in newly written code.

Reply:
Ok, I wasn't sure, I was just copying the name style of the logger it replaced